src/components/Body/index.tsx

```diff
@@ -20,22 +21,51 @@ const potentialPlays = [
   [1, 5, 9],
   [3, 5, 7],
 ];
+const horizontalArrays = [
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9],
+];
+const verticalArrays = [
+  [1, 4, 7],
+  [2, 5, 8],
+  [3, 6, 9],
+];
+const leftDiagonalArrays = [[1, 5, 9]];
+const rightDiagonalArrays = [[3, 5, 7]];
+
+const horizontalArraysRiskStyle = "rotate(180deg)";
```

Comment:
change all references to "risk" to "dash", risk is misleading, it means "Risco" ein portuguese, but in the sense of Danger, "arriscado"